src/transform/mod.rs

```diff
@@ -238,23 +238,15 @@ pub enum TxSet {
   // DCT only
   TX_SET_DCTONLY,
   // DCT + Identity only
-  TX_SET_DCT_IDTX,
+  TX_SET_INTER_3, // TX_SET_DCT_IDTX
   // Discrete Trig transforms w/o flip (4) + Identity (1)
-  TX_SET_DTT4_IDTX,
+  TX_SET_INTRA_2, // TX_SET_DTT4_IDTX
   // Discrete Trig transforms w/o flip (4) + Identity (1) + 1D Hor/vert DCT (2)
-  // for 16x16 only
-  TX_SET_DTT4_IDTX_1DDCT_16X16,
```

Comment:
@lu_zero, And three entries of TxSet enum was obsolete, i.e. discarded when av1 was finalized.

Reply:
Should be written in the commit message :)